meta-mel/conf/distro/mel.conf

```diff
@@ -177,6 +177,7 @@ BB_GIT_SHALLOW_REVS:pn-linux-mel ??= "${@'${LINUX_VERSION_TAG}' if '${LINUX_VERS
 
 require conf/distro/include/no-static-libs.inc
 require conf/distro/include/yocto-uninative.inc
+require conf/distro/include/security_flags.inc
```

Comment:
This looks a bit strict to me. Are we ready for this? Various packages might have issues due to the strictness of flags that this implies. We should at least keep a buffer in our estimates for the next product release if we're to follow this.

Reply:
Yeah, I mainly wanted to start the discussion on this. I can open a draft MR. It aligns with poky and other open source distros and I think is a step in the right direction, but as you say, could have impact if anything breaks as a result.

Reply:
Started an email thread and dropped the commit from this PR